mps-legacy-sync-plugin/src/main/java/org/modelix/model/mpsplugin/ModelServerConnection.kt

```diff
@@ -96,8 +96,11 @@ class ModelServerConnection @JvmOverloads constructor(baseUrl: String, providedH
         val workspaceTokenProvider: () -> String? = { InstanceJwtToken.token }
         val tokenProvider: (() -> String?)? =
             (if (ModelixConfigurationSystemProperties.executionMode == EModelixExecutionMode.PROJECTOR) workspaceTokenProvider else null)
-        val syncService = ApplicationManager.getApplication().getService(ModelSyncService::class.java)
-        val httpClient = providedHttpClient ?: syncService.httpClient
+        // Do not use ApplicationManager.getApplication().getService(ModelSyncService::class.java) here
```

Comment:
The code in the `ModelSyncService.init` should be executed somewhere from `ModelixSyncPluginInitializer` (maybe after the first project is registered) and not be part of the constructor.

Reply:
Done